moj-fe/app/Repositories/NewContentRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Repositories;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+
+class NewContentRepository {
+    protected $client;
+    protected $locale = '';
+
+    public function __construct() {
+        $this->client = new Client(array(
+            'base_uri' => config('app.api_uri'),
+            'timeout' => 60.0
```

Comment:
Is this not a configuration value somewhere and if not could we change it to one please?